packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -154,20 +155,65 @@ export default class SelectBox extends PureComponent {
     }
 
     componentDidUpdate() {
-        const listener = e => {
-            // Up and down arrow
-            if ([38, 40].indexOf(e.keyCode) > -1) {
-                e.preventDefault();
-            }
-        };
-
         if (this.state.isOpen) {
-            window.addEventListener('keydown', listener);
+            window.addEventListener('keydown', this.keydownListener);
         } else {
-            window.removeEventListener('keydown', listener);
+            window.removeEventListener('keydown', this.keydownListener);
         }
     }
 
+    keydownListener = debounce((e) => {
+        const UP_ARROW = 38;
```

Comment:
I think you could use import key definitions like this? https://www.npmjs.com/package/react-keydown#specify-multiple-keys-that-should-trigger-the-method